public/game-master/views/app-view.coffee

```diff
@@ -66,11 +68,16 @@ AppView = (context) ->
 
 					nxt.Element 'div',
 						nxt.Class 'controls-selector-banned-container'
-						nxt.Text 'banned chars'
-
-					nxt.Element 'div',
-						nxt.Class 'master-controls-puzzle'
-						nxt.Text 'HTML GOES HERE'
+						nxt.Text 'Banned chars'
+						nxt.Binding context.round_phase, (phase) ->
+							#should we each time re-create MatchRenderer? or use CSS to show/hide?
```

Comment:
For now - it's much easier to re-render the whole component. Would be nice to have a kind of pre-defined mechanism in `nexus` for such cases.